packages/lib/test/inputGen.test.ts

```diff
@@ -23,19 +24,23 @@ describe("merkle tree", () => {
       "044d9d03f3266f24777ac488f04ec579e1c4bea984398c9b98d99a9e31bc75ef0f13a19471a7297a6f2bf0126ed93d4c55b6e98ec286203e3d761c61922e3a4cda",
     ];
     const index = 2;
-    const poseidon = await buildPoseidonReference();
+    const poseidon = await buildPoseidon();
```

Comment:
Why did we move from reference to normal again?

Reply:
Or I guess buildPoseidonOpt

Reply:
Was just testing, reverted to buildPoseidonReference as we discussed